resources/js/components/DetailTabs.vue

```diff
@@ -119,6 +114,7 @@ export default {
     handleTabClick(tab, event) {
       tab.init = true;
       this.activeTab = tab.name;
+      this.$router.push({ query: { tab: tab.name }});
```

Comment:
Can you also add a replace here like in FormTabs.vue?

Reply:
I hope I understand it right, but I have done the change (and the merge of `master` into my `remember-current-tab branch`)